CODEOWNERS

```diff
@@ -1,2 +1,3 @@
+.github/*.  @erezrokah @hermanschaaf @yevgenypats
 cli/*       @yevgenypats
-plugins/source/* @cloudquery/cloudquery-opensource
\ No newline at end of file
+plugins/source/* @cloudquery/cloudquery-opensource
```

Comment:
```suggestion
/plugins/source/ @cloudquery/cloudquery-opensource
```

To quote the [CODEOWNERS example from the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):

```
# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat
```